tests/MongoDB.Bson.Tests/MongoDB.Bson.Tests.csproj

```diff
@@ -17,13 +17,25 @@
     <Description>MongoDB.Bson tests.</Description>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" PrivateAssets="All" Version="1.0.0" />
+  </ItemGroup>
+
   <PropertyGroup Condition="'$(Version)'==''">
     <Version>0.0.0-local</Version>
   </PropertyGroup>
 
   <PropertyGroup>
     <DefineConstants>TRACE</DefineConstants>
   </PropertyGroup>
+ 
+  <PropertyGroup>
+    <IsWindows Condition="$([MSBuild]::IsOSPlatform('Windows'))">true</IsWindows>
+  </PropertyGroup>
+  
+  <PropertyGroup Condition="'$(IsWindows)'=='true'">
+    <DefineConstants>WINDOWS</DefineConstants>
```

Comment:
similar in other places

Reply:
Instead of removing those lines I've just made sure the constants are not overridden. Like so: `<DefineConstants>$(DefineConstants);WINDOWS</DefineConstants>`